data/language/english_uk.txt

```diff
@@ -885,7 +885,7 @@ STR_0879    :Can't lower land here...
 STR_0880    :Can't raise land here...
 STR_0881    :Object in the way
 STR_0882    :Load Game
-STR_0883    :Save Game
+STR_0883    :Save Game as
```

Comment:
Why change this string when you're introducing the same one again later? Best to leave the string and just change the string id elsewhere, I think — less hassle for the other languages.